### PR TITLE
Solo: Restaurant Page v1.1

### PR DESCRIPTION
**Pull request by ybadmus**
This project comes after the introduction of the ES6 class keyword, ES6 modules, and webpack. The main goal of the project is to demonstrate understanding of the benefits of modularized code as well as the ability to set up a JavaScript application made of different modules using Webpack.

### CODE REVIEW

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Invalid submission :warning:
**Hello Yusif Badmus** 
Add linters to your project.
This Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready!
 
Happy coding :relaxed:
 
Oscar Andrés Russi Porras

File: package.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+    "name": "restaurant",
+    "version": "1.0.0",
+    "description": "understanding the benefits of modularized code\u001b[D\u001b[D\u001b[D\u001b[D\u001b[",
+    "main": "index.js",
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "build": "webpack",
+        "watch": "webpack --watch"
+    },
```
Comment (oloomoses) [inline package.json]: - [ ] Here is an example of how your script tag should look like
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack serve --mode development --open"
  },
```

**Review by oloomoses — CHANGES_REQUESTED** [review]
Hello @ybadmus,
### Highlights
- **The styling of this project is awesome :clap:**
- **Well done on the linter configuration**
- **most of the requirements are well covered :+1:**

### To improve
- [ ] `webpack-dev-server` should be added in the devDepencies. This will help us to run the server locally during development. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/devdep_requirement.png))
- [ ] the run commands in the script tag in `package.json` file should look the example below :point_down:
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack serve --mode development --open"
  },
``` 
This adds all the scripts needed to run the app ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/scripts_requirement.png))



**Kindly make the above changes before requesting another review**

Feel free to reach me on slack or leave a comment while mentioning me and I will  be happy to respond

_Slack: Oloo Moses_

**Review by mikenath223 — APPROVED** [review]
### CONGRATULATIONS  👏👏👏

This project meets the requirements for approval.

**STATUS: APPROVED ✅**


- [ ] OPTIONAL - Although, this should be a hard requirement it's been marked as optional since it wasn't mentioned in the previous review. Kindly ensure you use the exact eslint config rules as stated in this [link](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json). It helps to improve the professionalism of the project as these rules are shared by other students also.

Don't forget to fill the project completion form👍
Good luck with the next project.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

**Pull request merged**